alerting/src/main/kotlin/org/opensearch/alerting/resthandler/RestGetFindingsAction.kt

```diff
@@ -57,7 +58,9 @@ class RestGetFindingsAction : BaseRestHandler() {
 
         val getFindingsSearchRequest = GetFindingsRequest(
             findingID,
-            table
```

Comment:
Custom data sources are not exposed in rest layer. Let's limit the changes to transport layer.

Reply:
done